src/monsters.test.c

```diff
@@ -0,0 +1,273 @@
+#include <platform.h>
+#include <stdlib.h>
+
+#include <kernel/config.h>
+#include <kernel/faction.h>
+#include <kernel/item.h>
+#include <kernel/order.h>
+#include <kernel/race.h>
+#include <kernel/region.h>
+#include <kernel/terrain.h>
+#include <kernel/unit.h>
+
+#include "monster.h"
+#include "guard.h"
+#include "skill.h"
+
+#include <util/language.h>
+#include <util/language_struct.h>
```

Comment:
Dieser Header wird nur in language.h intern benutzt, und soll eigentlich nirgendwo anders gebraucht werden (die Internas dieser struct sind quasi private). Das ist nicht selbsterklärend, da will ich noch mal drüber nachdenken, und die Datei evtl. eliminieren.